log4j-nosql/src/main/java/org/apache/logging/log4j/nosql/appender/mongodb/MongoDbProvider.java

```diff
@@ -97,6 +103,8 @@ public String toString() {
     @PluginFactory
     public static MongoDbProvider createNoSqlProvider(
             @PluginAttribute("collectionName") final String collectionName,
+            @PluginAttribute(value = "capped", defaultBoolean = false) final Boolean isCapped,
```

Comment:
In order to preserve backwards compatibility with users who may be configuring things programmatically, we've been converting these factory methods into builders over time so that new attributes can be added. Do you think you could add a builder and deprecate this instead?